bmwqemu.pm

```diff
@@ -215,11 +215,17 @@ sub log_call {
     else {
         # key/value pairs
         my @result;
+        my $should_stash = 0;
+        my @tmp_array = @_;
+        while (my ($key, $value) = splice(@tmp_array, 0, 2)) {
+            $should_stash = 1 if ($key eq 'secret' && $value == 1);
```

Comment:
Isn't it sufficient if `$value` is a truthy value? The check `$value == 1` has the disadvantage of emitting a warning if `$value` isn't convertible to a number.

Reply:
at this point, yes. We are not doing parameter input validation, IMHO not the right place for that.